lib/tasks/peer_review.rake

```diff
@@ -0,0 +1,36 @@
+# NOTE: The assignment cannot be generated in the assignments rake task since
+# it causes errors to occur. I guess this is due to it not being populated
+# properly and something errors out. Therefore, all the peer review stuff is
+# done here, including but not limited to creating the PR assignment.
+
+namespace :db do
+  desc 'Create peer reviews and related data'
+  task :peer_reviews => :environment do
+    puts 'Creating A1 Peer Review and updating A1 to have the peer review'
+    a1 = Assignment.find_by(short_identifier: 'A1')
+    a1.has_peer_review = true  # Creates 'a1pr' via callback.
+    a1.save
+
+    a1pr = Assignment.find_by(short_identifier: a1.short_identifier + '_pr')
+    a1pr.clone_groupings_from(a1.id)
+
+    # TODO - Make a proper class for this so the code is clearer? Rather than some arbitrary list index?
+    # Populate a list of 'list of [group_id, result]'.
+    list_of_list_of_results_group_id = []
```

Comment:
It's not necessary to make this list.

Keep the second `each`, and inside that pick a random grouping from `a1.groupings`, and make the PeerReview.